src/nimblepkg/nimscriptapi.nim

```diff
@@ -178,10 +184,12 @@ template task*(name: untyped; description: string; body: untyped): untyped =
   proc `name Task`*() = body
 
   nimbleTasks.add astToStr(name)
+  if astToStr(name).len > nimbleTasksMaxNameLen:
+    nimbleTasksMaxNameLen = astToStr(name).len
+  nimbleTasksDescriptions.add description
```

Comment:
Maybe also turn `nimbleTasks` into a list of tuples to keep the names and descriptions together?

Reply:
Done.